opta/utils.py

```diff
@@ -172,3 +173,21 @@ def check_opta_file_exists(config_path: str) -> Literal[True]:
             )
         )
     return True
+
+
+def compare_datetime(date_time1: datetime, date_time2: datetime) -> int:
```

Comment:
why are you making this function? It's 2 lines which we only use once and you're adding extra features which we never use.

Reply:
Looked cleaner and helped with the test cases as well. Will try to remove it and update the test cases as well.

Reply:
Removed